pkgs/development/python-modules/pysam/default.nix

```diff
@@ -8,30 +8,33 @@
 , htslib
 , libdeflate
 , xz
-, pytest
+, pytestCheckHook
 , samtools
 , zlib
 }:
 
 buildPythonPackage rec {
   pname   = "pysam";
-  version = "0.20.0";
+  version = "0.21.0";
 
   # Fetching from GitHub instead of PyPi cause the 0.13 src release on PyPi is
   # missing some files which cause test failures.
   # Tracked at: https://github.com/pysam-developers/pysam/issues/616
   src = fetchFromGitHub {
     owner = "pysam-developers";
-    repo = "pysam";
+    repo = pname;
```

Comment:
We shouldn't change this as it is logical not fully correct.

Reply:
While it is the same value, it is semantically different and should carry the literal string.

Reply:
Reverted change.